### ISSUE TITLE
Recipients in request body of "mails/batch" endpoint email issue

### ISSUE THREAD

**Issue by awebartisan** [AhmadWaleed/mailservice#3]
From [Wiki](https://github.com/AhmadWaleed/mailservice/wiki/Send-Batch-Message), if you consider current json structure, Postman is not validating it.
Inside "recipients" array, having "email" as key for object is making it invalid json.

Valid json would be to include email address inside the object against "email" key.

`"recipients": [
    {
       "email": "john@example.com"
    }
]`

**Comment by awebartisan** [comment]
Its working with the structure I mentioned issue description. So this just needs to be updated in Wiki.

**Comment by AhmadWaleed** [comment]
Yes you're right but i am happy to go with invalidate json for ease.

Let's look with different angle:


```php
$payload =["john@example.com" => ["name" => "john"], "jane@example.com" => ["name" => "jane"]];
json_encode($payload)
```

From client side you simply pass valid encoded array and on api end just simply decode it

**Comment by AhmadWaleed** [comment]
My initial thought was only for php usage but i think you're right i will push a patch soon thanks for pointing out.

**Comment by AhmadWaleed** [comment]
@awebartisan can you please submit a pr for this?
